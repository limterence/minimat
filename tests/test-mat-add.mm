/* check-mat-add -- not same size */
int main() {
  matrix a;
  a = new matrix(4,2);
  printmat(a + a');     /* cannot add matrices of different sizes */
}

