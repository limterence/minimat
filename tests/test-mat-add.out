matrix row-column index out of bounds.  Exiting...
